docs/about.rst

```diff
@@ -168,6 +168,18 @@ the abstract base classes defined in ``parmoo.structs``.
 Glossary
 --------
 
+ * **Acquisition function:**
+   An *acquisition function* is our language for a family of scalarizing
+   functions, which can be used to specify targets on the Pareto front.
+   Acquisition functions may use objective scores, gradient values, and/or
+   uncertainty information in order to guide ParMOO's search for an
+   approximation to the Pareto front.
+
+    * **Ex.--** several common acquisition functions from the literature
+      include weighted sums (averages) of objective values, the epsilon
+      constraint method, and expected hypervolume improvement, which is
+      used for multiobjective Bayesian optimization.
```

Comment:
cannot each of these three example scalarizations be used in BO? Is there a benefit to making the hypervolume connection to BO? If not, but a MO BO connection is really desired, could perhaps say that these are common in the BO literature?

Reply:
deleted BO references, it was adding nothing of value here